src/main/java/robotbuilder/data/properties/ValuedParameterDescriptor.java

```diff
@@ -99,6 +102,8 @@ public boolean isValid() {
      */
     public boolean valueMatchesType() {
         switch (getType()) {
+            case "DoubleSupplier":
+                return true;
```

Comment:
This should probably have the same verification as String

Reply:
I added the Validate as string. I recall stubbing in the true because I didn't know if there was a way to validate a valid supplier. I also added a string not empty to make sure something is there. Either way though it will be up to the user to populate a valid string to satisfy the Double Supplier using one of the several different syntax.